libs/persona2vec_gcn/tests/test.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+
+import persona2vec_gcn as pvgcn
+from persona2vec_gcn.model import Persona2VecGCN
+from persona2vec_gcn.vgae import DeepVGAE
+import numpy as np
+import networkx as nx
+
+
+class TestPersona2VecGCN(unittest.TestCase):
+    def setUp(self):
+        self.G = nx.karate_club_graph()
+        self.A = nx.adjacency_matrix(self.G)
+
+    def test_embedding(self):
+        dimensions = 64
+        X = pvgcn.laplacian_eigenmap(self.G, dimensions=16)
+        model = Persona2VecGCN(self.G, X=X, lambd=0.1, dimensions=dimensions)
```

Comment:
> Add a test script for quality control
I encourage you to create a test script, and make sure that the program passes the test before closing the tree. This may appear to be cumbersome at first but, in my experience, this will substantially save our time. You can set up Github Action to automate the test.